test/conformance/runtime/envpropagation_test.go

```diff
@@ -92,6 +97,11 @@ func TestConfigsViaEnv(t *testing.T) {
 		}
 	})
 
+	if test.ServingFlags.OnlyRequiredAPI {
+		// Container.envFrom is not required by Knative Serving API Specification.
+		return
```

Comment:
Wondering why this one isn't a Skip like the others

Reply:
Changed. I was thinking about that there are two test cases in this test function so maybe just return instead of skipping one test case? Perhaps no differences.